packages/vue/src/components/molecules/SfRange/SfRange.stories.js

```diff
@@ -1,117 +1,136 @@
-      disabled: {
-        default: boolean("disabled", false, "Props"),
+import { SfRange } from "@storefront-ui/vue";
+export default {
+  title: "Components/Molecules/Range",
+  component: SfRange,
+  argTypes: {
+    value: {
+      control: {
+        type: "array",
       },
-      config: {
-        default: object("config",
-          {
-            start: [0, 1],
-            range: {
-              min: 0,
-              max: 10
-            },
-            step: 1,
-            connect: true,
-            direction: "ltr",
-            orientation: "horizontal",
-            behaviour: "tap-drag",
-            tooltips: true,
-            format: {
-              to: function (value) {
-                return new Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }).format(value)
-              },
-              from: function (value) {
-                const parsedValue = new Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }).formatToParts(value)
-                return parsedValue[0].value
-              }
-            },
-            keyboardSupport: true,
-          },
-          "Props"),
-      }
-    },
-    data() {
-      return {
-        value: [0, 1],
-      }
+      table: {
+        category: "Props",
+      },
+      defaultValue: {
+        summary: [0, 1],
+      },
+      description: "value",
     },
-    methods: {
-      alert(value) {
-        console.log("slider current value:", value);
+    disabled: {
+      control: "boolean",
+      table: {
+        category: "Props",
+      },
+      defaultValue: {
+        summary: false,
       },
+      description: "Disabled",
     },
-    components: { SfRange },
-    template:
-      `<div style="padding: 100px">
-			<SfRange
-        v-model="value"
-        :disabled="disabled"
-        :config="config"        
-				@change="alert"
-			/>  
-		</div>`,
-  }))
-  .add("One slider", () => ({
-    props: {
-      disabled: {
-        default: boolean("disabled", false, "Props"),
+    config: {
+      control: "object",
+      table: {
+        category: "Props",
       },
-      config: {
-        default: object("config",
-          {
-            start: 0,
-            range: {
-              min: 0,
-              max: 10
-            },
-            step: 1,
-            connect: true,
-            direction: "ltr",
-            orientation: "horizontal",
-            behaviour: "tap-drag",
-            tooltips: true,
-            format: {
-              to: function (value) {
-                return new Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }).format(value)
-              },
-              from: function (value) {
-                const parsedValue = new Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }).formatToParts(value)
-                return parsedValue[0].value
-              }
-            },
-            keyboardSupport: true,
+      defaultValue: {
+        start: [0,1],
```

Comment:
make it flat instead of nested object of props

Reply:
Do you mean to flatten it in the story, like did now. Or should it be changed in component props?